docs/self-managed/platform-deployment/helm-kubernetes/guides/multi-namespace-deployment.md

```diff
@@ -1,35 +1,123 @@
 ---
 id: multi-namespace-deployment
 title: "Multi-namespace deployment"
-description: "Deploy Camunda 8 Self-Managed across namespaces for better resources utilization and to reduce redundancy"
+description: "Deploy Camunda 8 Self-Managed across namespaces for better resource utilization and to reduce redundancy."
 ---
 
-You can deploy Camunda 8 Self-Managed across namespaces, where we have a single management deployment (Identity and Web Modeler), and multiple automation deployments (Zeebe, Operate, Tasklist, Optimize).
+Camunda 8 Self-Managed can be deployed across multiple namespaces. This configuration includes a single management deployment (comprising Console, Identity, and Web Modeler) and multiple automation deployments (including Zeebe, Operate, Tasklist, and Optimize).
 
-In this set up each namespace has its own Helm deployment independently, but each one uses a different values file based on if the deployment mode is `management` or `automation`.
+In this setup, each namespace is deployed independently using Helm. The deployments are differentiated by their deployment mode - `management` or `automation` - and each utilizes a distinct values file tailored to its mode.
 
-The following are three values files, one for the `managment` deployment and two for the `automation` deployment:
+In the following example, we use three Helm value files, one for the `management` deployment and two for the `automation` deployments:
+
+## Preparation
+
+If you plan to use Console in a multi-namespace setup, then you need first to get the release info config for all namespaces/deployments.
+
+Assuming all the values files have been created according to multi-deployment (next sections), run the following script to get the release info for all deployments.
```

Comment:
Here we say "next sections", but I would link to this section, or wonder why we don't have it before this content.

Reply:
@christinaausley I updated this page as Ahmed is on FTO. I moved this section to the bottom and rephrased descriptions.